src/test/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/BundleMapperTest.java

```diff
@@ -0,0 +1,44 @@
+package uk.nhs.digital.nhsconnect.lab.results.translator.mapper;
+
+import org.hl7.fhir.dstu3.model.Bundle;
+import org.hl7.fhir.dstu3.model.HumanName;
+import org.hl7.fhir.dstu3.model.Practitioner;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static uk.nhs.digital.nhsconnect.lab.results.fixtures.PathologyRecordFixtures.generatePathologyRecord;
+
+@ExtendWith(MockitoExtension.class)
+public class BundleMapperTest {
+
+    @Test
+    public void testMapMessageToPathologyRecord() {
```

Comment:
Should we restrict this test to the default visibility as JUnit 5 test classes and methods don't need to be public?

Reply:
`public` is default without it. I'll remove it as that's how the majority of tests are (though it's quite inconsistent)

Reply:
`public` is not the default visibility in java